share/doc/homebrew/El_Capitan_and_Homebrew.md

```diff
@@ -4,7 +4,7 @@ Part of the OS X 10.11/El Capitan changes is something called [System Integrity
 
 SIP prevents you from writing to many system directories such as `/usr`, `/System` & `/bin`, regardless of whether or not you are root. The Apple keynote is [here](https://developer.apple.com/videos/wwdc/2015/?id=706) if you'd like to learn more.
 
-One of the implications of SIP is that you cannot simply create `/usr/local` if you have removed it. However, as noted in the keynote, Apple is leaving `/usr/local` open for developers to use, so Homebrew can still be used as expected.
+One of the implications of SIP was that you could not simply create `/usr/local` if you had removed it. This issue was fixed with the OS X 10.11.0 release. However, as noted in the keynote, Apple is leaving `/usr/local` open for developers to use, so Homebrew can still be used as expected.
```

Comment:
This is actually false. 10.11.0 full installer and 10.11.1 full installer both install version 10.0 of the compatibility bundle. Maybe 10.11.2 will come with 12.0 preinstalled, but that is not yet the case.

At least for now, [com.apple.pkg.SystemIntegrityProtectionConfig.14U2076] (http://swcdn.apple.com/content/downloads/53/38/031-40358/y7sn2l7yvfxb73myuiufqin8fp2sqyetvl/SystemIntegrityProtectionConfig.pkg) must have been installed one or another, regardless of whether it happened automatically or manually.